e2e/package.json

```diff
@@ -13,16 +13,16 @@
     "@types/mocha": "^5.2.6",
     "@types/node": "^11.13.4",
     "@types/selenium-webdriver": "^3.0.16",
-    "chai": "^4.2.0",
+    "axios": "^0.17.1",
+    "chai": "^4.1.2",
     "chromedriver": "^2.46.0",
     "mocha": "^6.1.4",
     "selenium-webdriver": "^3.6.0",
-    "ts-node": "^8.0.3",
-    "typed-rest-client": "^1.2.0",
-    "typescript": "^3.4.3"
+    "ts-node": "^8.1.0",
```

Comment:
ts-node@8.1.0 is still `awaiting` now. https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19663     I think no fix is required. Just waiting until fixed CQ.

Reply:
It was passed CQ check https://dev.eclipse.org/ipzilla/show_bug.cgi?id=19663. So I mark this conversation resolved.

Reply:
Thank you for your attention.